bin/varnishd/cache/cache_range.c

```diff
@@ -180,13 +181,66 @@ vrg_dorange(struct req *req, const char *r, void **priv)
 	return (NULL);
 }
 
+/*
+ * return 1 if range should be observed, based on if-range value
+ * if-range can either be a date or an ETag [RFC7233 3.1 p8]
```

Comment:
it actually is section 3.2 https://tools.ietf.org/html/rfc7233#section-3.2

Reply:
indeed, fixed, thank you